samples/nrf5340/extxip_smp_svr/sysbuild_no_network_core_directxip.conf

```diff
@@ -6,6 +6,7 @@
 
 SB_CONFIG_BOOTLOADER_MCUBOOT=y
 SB_CONFIG_BOOT_SIGNATURE_TYPE_ECDSA_P256=y
+SB_CONFIG_MCUBOOT_MODE_OVERWRITE_ONLY=y
```

Comment:
If you set it then `SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP` will not be applied (see https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/CMakeLists.txt#L191)

Reply:
Good catch, fixed